# Assignment 3: Exercises 6.1, 7.1, 8.1
Aaron Ou | 2018-03-06

### 6.1 a:

<!--(a) Estimate the minimum detectable effect for a two sided comparison of means with sample size of 440,
standard deviation of 10.2, significance level of 0.05, and power of 80%. What happens to the MDE if you increase
the sample size?
-->

In [1]:
# n = 440, sd = 10.2, sig.level = 0.05, power = 80%
power.t.test(n = 440, delta = NULL, sd = 10.2, sig.level = 0.05, power = 0.8,
             type = "two.sample",
             alternative = "two.sided",
             strict = FALSE)


     Two-sample t test power calculation 

              n = 440
          delta = 1.92873
             sd = 10.2
      sig.level = 0.05
          power = 0.8
    alternative = two.sided

NOTE: n is number in *each* group


If the sample size increases, the MDE will decrease.

### 6.1 b:

<!--
(b) Estimate the statistical power for a two sided comparison of means test with significance level of 0.10, sample
size of 20500, and standard deviation of 1010, and MDE of 250. What happens to the statistical power if you
increase the standard deviation?
-->

In [2]:
# n = 20500, sig.level  = 0.1, sd = 1010, MDE = 250
power.t.test(n = 20500, delta = 250, sd = 1010, sig.level = 0.1, power = NULL,
             type = "two.sample",
             alternative = "two.sided",
             strict = FALSE)


     Two-sample t test power calculation 

              n = 20500
          delta = 250
             sd = 1010
      sig.level = 0.1
          power = 1
    alternative = two.sided

NOTE: n is number in *each* group


If the standard deviation increases, statistical power will decrease.

### 6.1 c:

<!--
(c) Estimate the required sample size for significance level of 0.05, power of 80% standard deviation of 0.65 and
MDE of .03. What happens to the required sample size when you increase the power?
-->

In [3]:
# n = NULL, sig.level = 0.05, power = 0.8, sd = 0.65, MDE = .03
power.t.test(n = NULL, sig.level = 0.05, power = 0.8, sd = 0.65, delta = 0.03,
             type = "two.sample",
             alternative = "two.sided",
             strict = FALSE)


     Two-sample t test power calculation 

              n = 7370.186
          delta = 0.03
             sd = 0.65
      sig.level = 0.05
          power = 0.8
    alternative = two.sided

NOTE: n is number in *each* group


A required sample size of 7370.
If the power increases, the required sample size will have to increase too to maintain the same MDE.